spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+
```

Comment:
Inside the `user_controller` spec?
Again, test writing is something I'm going to need some help with.